tutorials/deutsch_jozsa/deutsch_jozsa.ipynb

```diff
@@ -0,0 +1,393 @@
+{
```

Comment:
<p>Overall, it looks good!</p><p> </p><p>I can't view the image. You can either add it to the same folder in the PR or, preferably, add it via a link from our AWS server, which you probably don’t have access to (same for me). It would be best to confirm this with Eden.</p><p> </p><p>Additionally, I usually take a more "to-the-point" approach with less storytelling. We should discuss with Eden to ensure the level of uniformity we wish to achieve in our materials.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I would also add more details on the quantum advantage. </p><p>What is the idea of classical solution?</p><p>What is the edge given by DJ algorithm? (single query vs n queries)</p><p> </p>

Reply:
About the image, how do you use it from the AWS bucket? I copied the link of the JPEG from the bucket as a URL and used the img HTML tag